zebra-state/src/constants.rs

```diff
@@ -46,7 +46,7 @@ pub const STATE_DATABASE_KIND: &str = "state";
 ///
 /// Instead of using this constant directly, use [`constants::state_database_format_version_in_code()`]
 /// or [`config::database_format_version_on_disk()`] to get the full semantic format version.
-const DATABASE_FORMAT_VERSION: u64 = 25;
+const DATABASE_FORMAT_VERSION: u64 = 26;
```

Comment:
Could we bump the minor db version and store the new `ValueBalance` disk format with a different key?

Reply:
Bumping the database version seems like the right solution to me because it doesn't require workarounds such as storing the new `ValueBalance` under a new key. We also did previous database format upgrades such as pruning the note commitment trees where previous Zebra versions couldn't read the new format anymore.

Reply:
> it doesn't require workarounds such as storing the new ValueBalance under a new key

We could make it temporary by adding a downgrade for a future database format change that moves it back to the `()` key, it would be nice to avoid requiring zebrad to sync from scratch for such a small format change.